### PR TITLE
CLI Search updated to use API v1. API url updated AEA-850

### PR DESCRIPTION
**Pull request by panasevychol**
## Proposed changes

CLI Search updated to use API v1. API url updated.
#1635 

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by DavidMinarsch — APPROVED** [review]
LGTM; what about loading the next page?

File: aea/cli/search.py [hunk 1]
```diff
@@ -62,47 +62,52 @@ def search(click_context, local):
...
+def skills(ctx: Context, query, page):
     """Search for Skills."""
     item_type = "skill"
-    _output_search_results(item_type, search_items(ctx, item_type, query))
+    _output_search_results(item_type, *search_items(ctx, item_type, query, page), page)
 
 
 @search.command()
 @click.option("--query", default="", help="Query string to search Agents by name.")
+@click.option("--page", type=int, default=1, help="Page number to display.")
```
Comment (marcofavorito) [inline aea/cli/search.py:105]: you could consider creating a composed annotator to avoid code duplication.

Comment (panasevychol) [reply aea/cli/search.py:105]: @marcofavorito how can I do that?
And actually that's not a part of this PR as we have already `--query` described for all commands separately. Would be nice if you will create a ticket and I will do optimization for those commands in other PR.

Comment (marcofavorito) [reply aea/cli/search.py:105]: basically, just define a function with all the decorators in one place (I'd say at the top of the module). But you're right, it's out of the scope for this PR.

**Comment by marcofavorito** [comment]
LGTM, but the failing tests need to be fixed

**Review by 5A11 — APPROVED** [review]
LGTM

**Review by marcofavorito — APPROVED** [review]
LGTM

**Pull request merged**